src/sentry/web/frontend/accounts.py

```diff
@@ -119,11 +120,12 @@ def start_confirm_email(request):
     has_unverified_emails = request.user.has_unverified_emails()
     if has_unverified_emails:
         request.user.send_confirm_emails()
-        msg = _('A verification email has been sent to %s.') % request.user.email
+        unverified_emails = request.user.get_unverified_emails()
+        msg = _('A verification email has been sent to %s.') % (', ').join([email.email for email in unverified_emails])
```

Comment:
you can actually avoid the list comprehension here and just let it operate on the generator (in ', '.join())